instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v1_4/JedisDbAttributesExtractor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jedis.v1_4;
+
+import io.opentelemetry.instrumentation.api.db.RedisCommandSanitizer;
+import io.opentelemetry.instrumentation.api.instrumenter.db.DbAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import redis.clients.jedis.Connection;
+
+final class JedisDbAttributesExtractor extends DbAttributesExtractor<JedisRequest> {
+  @Override
+  protected String system(JedisRequest request) {
+    return SemanticAttributes.DbSystemValues.REDIS;
+  }
+
+  @Override
+  @Nullable
+  protected String user(JedisRequest request) {
+    return null;
+  }
+
+  @Override
+  protected String name(JedisRequest request) {
+    return null;
+  }
+
+  @Override
+  protected String connectionString(JedisRequest request) {
+    Connection connection = request.getConnection();
+    return connection.getHost() + ":" + connection.getPort();
```

Comment:
Since `JedisRequest` is our own class how about inlining these two business logic into it?